charts/spire/charts/spiffe-csi-driver/README.md

```diff
@@ -23,7 +26,7 @@ A Helm chart to install the SPIFFE CSI driver.
 | nodeDriverRegistrar.image.repository | string | `"sig-storage/csi-node-driver-registrar"` |  |
 | nodeDriverRegistrar.image.version | string | `"v2.6.2"` |  |
 | nodeDriverRegistrar.resources | object | `{}` |  |
-| nodeSelector."kubernetes.io/arch" | string | `"amd64"` |  |
+| nodeSelector | object | `{}` |  |
```

Comment:
Do we still need a nodeSelector configurable?

Reply:
Yes.

Reply:
@faisal-memon yes you will need it if you prefer to install spire-agent on specific nodes only. E.g. nodes where you want to use spire for workloads. That can save some resources on other nodes that don't run spire enabled workloads. E.g. we have some gpu nodes that don't require spire.